test/onigumo_test.exs

```diff
@@ -39,14 +39,13 @@ defmodule OnigumoTest do
     Enum.map(@urls, &assert_downloaded(&1, tmp_dir))
   end
 
-  @tag :tmp_dir
-  test("load URLs from a file", %{tmp_dir: tmp_dir}) do
-    input_path_env = Application.get_env(:onigumo, :input_path)
-    input_path_tmp = Path.join(tmp_dir, input_path_env)
-
-    for slice <- @slices do
-      input_urls = Enum.slice(@urls, slice)
+  for slice <- @slices do
+    @tag :tmp_dir
+    test("load URLs #{inspect(slice)} from a file", %{tmp_dir: tmp_dir}) do
+      input_urls = Enum.slice(@urls, unquote(Macro.escape(slice)))
```

Comment:
I resorted to putting the whole complex expression `unquote(Macro.escape(slice))` inline instead of assigning its result to an intermediary variable. I understood it as a wrapper around the name to use the variable and tried to hide the logic behind it. If the variable appeared in multiple places, I might have chosen differently.

```suggestion
      slice_param = unquote(Macro.escape(slice))
      input_urls = Enum.slice(@urls, slice_param)
```